app/styles/ui/_diff.scss

```diff
@@ -286,17 +286,36 @@
   }
 }
 
-.image-diff-two-up {
+.image-diff-container {
   display: flex;
   flex: 1;
   justify-content: center;
   align-items: center;
+  flex-direction: column;
 
-  .image-diff-previous {
-    margin-right: 7.5px;
+  .image-diff-two-up {
+    display: flex;
+    align-items: center;
+
+    .image-diff-previous {
+      margin-right: 7.5px;
```

Comment:
Thought on replacing this with `margin-right: var(--spacing);` instead? This will create the standard spacing between the image diffs.

Reply:
See https://github.com/desktop/desktop/pull/4403#discussion_r187312119